src/admin-views/page.php

```diff
@@ -108,6 +108,23 @@
                     <label for="markVirtual">Make them Virtual events.</label>
                 </td>
             </tr>
+        <?php endif; ?>
+        <?php if( class_exists( 'Tribe__Events__Pro__Main' ) ) : ?>
+            <tr class="tribe-dependent" data-depends="#numEvents" data-condition-not="0" style="background-color: whitesmoke">
+                <td>
+                    <input type="checkbox" id='isRecurring' name='tribe-ext-test-data-generator[events][recurring]'/>
+                    <label for="isRecurring">Make them Recurring.</label>
+                </td>
+                <td class="tribe-dependent" data-depends="#isRecurring" data-condition-is-checked="true" style="background-color: whitesmoke">
+                    <label for="recurringType">Recurrence: </label>
```

Comment:
Late to the game, but products coding standards require `snake_case`, not `camelCase`. Might want to consider changing these at some point?

Reply:
I'll fix this in an upcoming enhancement of that page, I'll add some other stuff.

Reply:
Thank you @Camwyn ! This is now done! 🎉